dist/riff.js

```diff
@@ -11,6 +12,36 @@ const repeat = (str, count = 1) => {
   return replacedStr;
 };
 
+const scribbleClipToMidiSteps = scribbleClip => {
```

Comment:
This is fantastic. Once we merge in this change I ll move it to Scribbletune's max method so that users can invoke it while creating the apiClip dict directly.

Reply:
(you are free to do it too but i didnt want to make an assumption.. also you are already doing quite a lot!)

Reply:
Hey, no worries @walmik , it's my way of giving back, since Scribbletune gave me such a great opportunity to practice music theory and javascript at the same time :)

But I'm definitely not making any promises with this one. I'm not that familiar with inner workings of Scribbletune, so I dont know how long it would take me.

Reply:
Perfectly understandable.. actually lemme take a stab at it in Scribbletune since it ll need some back n forth with scribbleformax.

> since Scribbletune gave me such a great opportunity to practice music theory and javascript at the same time

it did that for me too :)